docs/source/testing.rst

```diff
@@ -0,0 +1,7 @@
+torch.testing
+=============
+
```

Comment:
Do we need a disclaimer about the beta status here?

Reply:
Yes!

We should probably steal something like what torch.linalg has:

"Warning! This module is in BETA. New functions are still being added, and existing functions may change in future PyTorch releases."